.gitignore

```diff
@@ -66,6 +66,7 @@ build/
 cpp/build/
 cpp/include/cudf/ipc_generated/*.h
 cpp/thirdparty/googletest/
+cpp/cmake-build-debug
```

Comment:
This folder is specific to you.  Please use cpp/Debug, which is already .gitignored.